examples/cognitoidp/userpool.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cognitoidp.aws.upbound.io/v1beta1
+kind: UserPool
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cognitoidp/v1beta1/userpool
```

Comment:
I belive we need manual intervention for this resource.

Reply:
I have added manual intervention.